docs/en_US/Release_v1.0.md

```diff
@@ -0,0 +1,33 @@
+
+<p align="center">
+<img src=".././img/release-1-title-1.png" width="100%" />
+</p>
+
+
+From September, 2018 to September, 2019, We are still moving on …
+
+
+ **Great news!**&nbsp;&nbsp;With the tag of **Scalability** and **Ease of Use** , NNI v1.0 is comming. Based on the various types of [Tuning Algorithms](./Tuner/BuiltinTuner.md), NNI has supported the [Neural Architecture Search](docs/en_US/AdvancedFeature/GeneralNasInterfaces.md) and  Auto-Feature generator & selection,which is an exciting news for algorithmic engineers, besides these, NNI v1.0 has made many improvements in the optimization of tuning algorithm,  [WebUI's simplicity and intuition](./Tutorial/WebUI.md) and [platform diversification](./TrainingService/SupportTrainingService.md). NNI has grown into a more intelligent automated machine learning(AutoML) toolkit.
+ 
+
+
+<br/>
+<br/>
+<br/>
+<p align="center">
+<img src=".././img/nni-1.png" width="80%" />
+</p>
+
+<br />
+<br />
+
+<p align="center">
+<img src=".././img/release-1-title-2.png" width="100%" />
+</p>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step one**：Start with the [Tutorial Document](./Tutorial/Installation.md),and install NNI v1.0 first.<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step two**：Find a " Hello world example", follow the [Tutorial Documentation](./Tutorial/QuickStart.md) and have a Quick Start. <br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step three**：Get familiar with the [WebUI Tutorial](./Tutorial/WebUI.md) and let NNI better assists with your tuning tour.<br>
+
+The fully automated tool greatly improves the efficiency of the tuning process. For more detail about the 1.0 updates, you can refer to [Release 1.0](https://github.com/microsoft/nni/releases) . More of our advance plan, you can refer to our [Roadmap](https://github.com/microsoft/nni/wiki/Roadmap).Besides, we also welcome more and more contributors to join us, there are many ways to participate, please refer to [How to contribute](./Tutorial/Contributing.md
```

Comment:
Extra space before `. More of our advance plan`. Missing space after the link of Roadmap.

Reply:
ok ,all fixed